### PR TITLE
Updating CODEOWNERS with currently active members

### PR DESCRIPTION
**Pull request by mjmckay**

### CODE REVIEW

**Comment by fdegir** [comment]
+1

Thanks for the update!

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # These owners will be the default owners for everything in
 # the repo. People listed below will be requested for review
 # when someone opens a pull request.
-* @aglover @bobcatfish @ahpook @fdegir @fuqiao123 @MarckK @pritianka @zxiiro @tracymiranda @wavell @agrimmer @chhsia0 @ravilach
+* @mjmckay @amfred @e-backmark-ericsson @olivernocon @afrittoli @thedevelopnik @rakhbari @bobcatfish @fdegir @MarckK @zxiiro @tracymiranda @agrimmer
```
Comment (justinabrahms) [inline .github/CODEOWNERS:4]:
```suggestion
* @mjmckay @amfred @e-backmark-ericsson @olivernocon @afrittoli @thedevelopnik @rakhbari @bobcatfish @fdegir @MarckK @zxiiro @tracymiranda @agrimmer @justinabrahms
```

**Comment by mjmckay** [comment]
I've added a fix to the meeting time for the season and a correction to the hackmd link to this pull request.

**Comment by fdegir** [comment]
> @fdegir We need to add Justin as a maintainer to get rid of the error in the codeowners file

Done. Justin should've received an invite now.

**Pull request merged**